packages/tools/src/utilities/annotationFrameRange.ts

```diff
@@ -0,0 +1,84 @@
+import { triggerEvent, eventTarget } from '@cornerstonejs/core';
+import Events from '../enums/Events';
+import { Annotation } from '../types';
+
+/**
+ * This class handles the annotation frame range values for multiframes.
+ * Mostly used for the Video viewport, it allows references to
+ * a range of frame values.
+ *
+ */
+
```

Comment:
This should be exactly on top of the class name so that it gets picked up by api extractor for api docs